### PR TITLE
Update Gov params and update links

### PR DESCRIPTION
**Pull request by kollegian**
## What is the purpose of the change?

Updates gov params as queried from pacific 1
Updates some links 
Removes some todo

### CODE REVIEW

**Review by Cordtus — APPROVED** [review]

**Review by cordt-sei — APPROVED** [review]
Oops let me use the correct account this time. Double approved, thank you.

File: pages/dev-frontend-dapps.mdx [hunk 1]
```diff
@@ -9,11 +9,11 @@ Connecting to wallets is a crucial step in developing dApps. Here are some recom
 ### EVM and EVM RPC dApps:
 
 - **Wagmi**: A React-based library for Ethereum dApps that simplifies wallet connection and interaction. Provides hooks for interacting with Ethereum wallets and contracts for use with modern frontend libraries and frameworks.
-  - [Wagmi Documentation](https://wagmi.sh/docs)
+  - [Wagmi Documentation](https://docs.wagmi.com/)
```
Comment (besated) [inline pages/dev-frontend-dapps.mdx]: i think this is the wrong link - we want wagmi https://wagmi.sh/

**Review by besated — CHANGES_REQUESTED** [review]

File: pages/dev-frontend-dapps.mdx (hunk 1 above)
Comment (kollegian) [reply pages/dev-frontend-dapps.mdx]: You are def right, updated the link to https://wagmi.sh/

**Review by cordt-sei — APPROVED** [review]
looks good

**Review by besated — APPROVED** [review]

**Pull request merged**